Content.Client/Chat/TypingIndicator/TypingIndicatorSystem.cs

```diff
@@ -82,4 +83,21 @@ private void OnShowTypingChanged(bool showTyping)
             ClientUpdateTyping(false);
         }
     }
+
+    /// <summary>
+    /// DeltaV: Client can type with alternate indicators
+    /// </summary>
+    /// <param name="protoId">The TypingIndicator to show in place of the normal TypingIndicator</param>
+    public void ClientAlternateTyping(ProtoId<TypingIndicatorPrototype> protoId)
+    {
+        if (!_cfg.GetCVar(CCVars.ChatShowTypingIndicator))
+            return;
```

Comment:
Subs.CVar instead of getting it every time

Reply:
Is this something that should be changed in the upstream service? It does actually sub to it but then never actually saves the value